frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -454,6 +455,9 @@ export const experimentLogic = kea<experimentLogicType>([
         setExperimentType: async ({ type }) => {
             actions.setExperiment({ type: type })
         },
+        setExperimentFeatureFlagKeyFromName: async () => {
+            actions.setExperiment({ feature_flag_key: values.dynamicFeatureFlagKey })
```

Comment:
You can simply call `actions.setExperiment` directly in your UI component, rather than creating a new `setExperimentFeatureFlagKeyFromName` action + listener (same goes for the `setExperimentType` above which I had overlooked in Phani's earlier PR.

Reply:
Removed `setExperimentFeatureFlagKeyFromName` with 33069c15fd